build.gradle.kts

```diff
@@ -132,6 +138,7 @@ val copyMavenScripts by tasks.registering(Copy::class) {
 
     // local variable required for configuration cache compatibility
     // https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:not_yet_implemented:accessing_top_level_at_execution
+    val buildScanSummaryVersion = buildScanSummaryVersion
```

Comment:
Swap with the line below and then add a line break.

Reply:
Done in https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/487/commits/b9cb2f603e6c14a588d1b864c12ee0459872037f